Modix/ModixBot.cs

```diff
@@ -137,6 +138,7 @@ public void LoadConfig()
             _map.AddSingleton(_config);
             _map.AddScoped<IQuoteService, QuoteService>();
             _map.AddSingleton<CodePasteService>();
+            _map.AddSingleton<IHttpClient, HttpClient>();
```

Comment:
I'm pretty sure this usage is just asking for problems. Things like BaseUrl or DefaultRequestHeaders would be shared between different services, which could cause random weird breaks.

Should probably encapsulate that in the cat service, and inject the cat service instead.